SCClassLibrary/Common/Core/Boolean.sc

```diff
@@ -18,9 +18,13 @@ Boolean {
 	keywordWarnings {
 		// turn on/off warnings if a keyword argument is not found
 		_KeywordError
+		^this.primitiveFailed
+	}
+	trace {
+		// this is only available in a special debugging version of the app
```

Comment:
Ok. I would much rather have appreciated this response than posting an article link without comment, which made me feel like I was being dismissed even though you may have intended an entirely different effect. Something I've learned over the last few weeks is that it's healthier to actively redirect conversations than to shut them down or passively defer them. The reason I brought this up here is because we have no precedent for starting or maintaining these conversations. So, it is easy to take a request that we "discuss this somewhere else" as a similar kind of dismissal. But obviously, our conversation about my initial comment has generated a significant amount of noise on this PR, so your point is well-taken! Maybe we'll have a chance to discuss this on Saturday. :)